Removed quotes closes #2827

myleshorton opened the pull request.

myleshorton commented:
I'm just going to merge this in the interest of getting this release out, but please have a look @uaalto and @xiam

uaalto commented:
I apologize, I introduced this. I have had just the opposite experience with quotes in Makefile, so I added them to avoid issues.

myleshorton commented:
No worries!
On Jul 27, 2015 2:32 PM, "Ulysses Aalto" <notifications@github.com> wrote:

> I apologize, I introduced this. I have had just the opposite experience
> with quotes in Makefile, so I added them to avoid issues.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/lantern/pull/2828#issuecomment-125348734>.
>
